LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -339,5 +443,32 @@ public enum ELoggingLevel : byte
             /// </summary>
             Trace = 0
         }
+
+        /// <summary>
+        /// Flags for output Sources
+        /// </summary>
+        [Flags]
+        public enum EOutputSource : byte
+        {
+            /// <summary>
+            /// No Logging output at all
+            /// </summary>
+            None = 0,
+
+            /// <summary>
+            /// Write To Console
+            /// </summary>
+            Console = 1,
+
+            /// <summary>
+            /// Write into File
+            /// </summary>
+            File = 2,
+
+            /// <summary>
+            /// Write into Costume definde Streams
+            /// </summary>
+            Custom = 4
```

Comment:
also: you may want to just have multiple custom steams, so i'd prefer to get rid of that here and have `AddCustomStreams` as the only option to add Custom streams, right?